### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by Trictonicmp**
#### Capstone project presentation
I decided to name it Walist, a mix between wallet and list, walist helps users keep track of their expenses and see where they are spending their money, walist allows users to understand their expenses, this gives users the opportunity to make changes to their lifestyle according to their criteria.

Here you can find the video presentation:
[Presentation video](https://drive.google.com/file/d/1UEDGbrU69fbnRQdkGSdOOkekSuWqutLz/view?usp=sharing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,68 @@
...
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@Trictonicmp](https://github.com/Trictonicmp)
+- LinkedIn: [Amaury Permer](https://linkedin.com/in/amaury-permer)
 
-👤 **Author2**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (rloterh) [inline README.md:66]: - [x] It's a nice and smart thing to give appropriate credit to **[Gregoire Vella on Behance](https://www.behance.net/gregoirevella)**, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.

File: README.md [hunk 2]
```diff
@@ -70,6 +82,6 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (rloterh) [inline README.md]: - [x] OPTIONAL
Kindly endeavor to remove instruction text beneath License section

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Amaury, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Properly deployed project
- Implemented required pages
- App is running as intended
- Form validation nicely implemented
- Good rspec unit test coverage



## Required Changes ♻️
- [x] Good job with project presentation, however, you missed some steps in your video presentation. Your video presentation only showed functional aspect of your app and failed to highlight any portion of your code. Kindly ensure that the video contains a highlight about a piece of the code that you find interesting or that you were proud of making.

_Kindly also check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Trictonicmp** [comment]
hi @rloterh  Thank you for your kind review, I Added the acknowledgment and deleted the note on the README footer, I'll ask for another review!

**Comment by rloterh** [comment]
> hi @rloterh Thank you for your kind review, I Added the acknowledgment and deleted the note on the README footer, I'll ask for another review!

Nice job, however, kindly also improve upon your video presentation. Kindly check in the review remarks for the same.

Cheers!

**Review by julie-ify — APPROVED** [review]
Hi @Trictonicmp,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**